csrc/device_lower/analysis/circular_buffer.h

```diff
@@ -136,8 +148,12 @@ class CircularBufferInfo {
   //! iterdomains.
   std::unordered_map<IterDomain*, std::unordered_set<const TensorView*>>
       circular_buffer_tvs_;
-  //! True if the fusion has warp specialized circular buffer
-  bool has_warp_sepcialized_ = false;
+
+  //! Number of computation warp groups
```

Comment:
We already use multiple compute warp groups in the matmul kernel. This isn't correct for distinishing ping-pong from cooperative.

Reply:
This is only added for normalization. For matmul, can we add a new function or revise it?

Reply:
This PR is insufficient for ping-pong matmuls, so I reimplemented in https://github.com/NVIDIA/Fuser/pull/4460.